packages/design-system/src/components/popup/index.js

```diff
@@ -20,98 +20,124 @@
 import styled from 'styled-components';
 import {
   useLayoutEffect,
+  useEffect,
   useCallback,
   useState,
   useRef,
   useResizeEffect,
   createPortal,
-  useEffect,
 } from '@googleforcreators/react';
 import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
-import { themeHelpers } from '../../theme';
 import { noop } from '../../utils';
 import { getTransforms, getOffset } from './utils';
 import { PLACEMENT } from './constants';
-
-// TODO scrollbar update, commented out until design updates are done
-const DEFAULT_POPUP_Z_INDEX = 11;
-
+const DEFAULT_TOPOFFSET = 0;
 const Container = styled.div.attrs(
   ({
     $offset: { x, y, width, height },
     fillWidth,
     fillHeight,
     placement,
-    zIndex,
+    isRTL,
+    invisible,
   }) => ({
     style: {
-      transform: `translate(${x}px, ${y}px) ${getTransforms(placement)}`,
+      transform: `translate(${x}px, ${y}px) ${getTransforms(placement, isRTL)}`,
       ...(fillWidth ? { width: `${width}px` } : {}),
       ...(fillHeight ? { height: `${height}px` } : {}),
-      zIndex,
+      ...(invisible ? { visibility: `hidden` } : {}),
     },
   })
 )`
   /*! @noflip */
   left: 0px;
   top: 0px;
   position: fixed;
-
-  ${themeHelpers.scrollbarCSS};
+  z-index: 2;
+  ${({ noOverFlow }) => (noOverFlow ? `` : `overflow-y: auto;`)};
```

Comment:
Nit for consistency: 
```suggestion
  ${({ noOverFlow }) => (noOverFlow ? '' : 'overflow-y: auto;')};
```

Reply:
I think even ` ${({ noOverFlow }) => noOverFlow && 'overflow-y: auto;'};` works